<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>828</width>
                <height>517</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>分组统计</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
                <widget class="QFrame" name="frame_4">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                            <horstretch>1</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                            <widget class="QLabel" name="label_3">
                                <property name="text">
                                    <string>拖拽到右侧</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QListWidget" name="listWidget"/>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QFrame" name="frame_3">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                            <horstretch>1</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <widget class="QFrame" name="frame">
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>80</height>
                                    </size>
                                </property>
                                <property name="frameShape">
                                    <enum>QFrame::StyledPanel</enum>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Raised</enum>
                                </property>
                                <layout class="QHBoxLayout" name="horizontalLayout_3">
                                    <item>
                                        <widget class="QLabel" name="label">
                                            <property name="text">
                                                <string>分组</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QWidget" name="widget_group" native="true">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>300</width>
                                                    <height>0</height>
                                                </size>
                                            </property>
                                            <property name="acceptDrops">
                                                <bool>true</bool>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true">background-color: #e7e7e7;</string>
                                            </property>
                                            <layout class="QHBoxLayout" name="horizontalLayout">
                                                <property name="spacing">
                                                    <number>7</number>
                                                </property>
                                                <property name="leftMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="topMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="rightMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="bottomMargin">
                                                    <number>0</number>
                                                </property>
                                            </layout>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <widget class="QFrame" name="frame_2">
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>80</height>
                                    </size>
                                </property>
                                <property name="frameShape">
                                    <enum>QFrame::StyledPanel</enum>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Raised</enum>
                                </property>
                                <layout class="QHBoxLayout" name="horizontalLayout_4">
                                    <item>
                                        <widget class="QLabel" name="label_2">
                                            <property name="text">
                                                <string>汇总</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QWidget" name="widget_stat" native="true">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>300</width>
                                                    <height>0</height>
                                                </size>
                                            </property>
                                            <property name="acceptDrops">
                                                <bool>true</bool>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true">background-color: rgb(231, 231, 231);</string>
                                            </property>
                                            <layout class="QHBoxLayout" name="horizontalLayout_2">
                                                <property name="spacing">
                                                    <number>7</number>
                                                </property>
                                                <property name="leftMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="topMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="rightMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="bottomMargin">
                                                    <number>0</number>
                                                </property>
                                            </layout>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="btn_ok">
                                <property name="styleSheet">
                                    <string notr="true">color: rgb(255, 255, 255); background-color: rgb(28, 177,
                                        245);
                                    </string>
                                </property>
                                <property name="text">
                                    <string>OK</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
